### PR TITLE
Change status bar color when contents scrolled in session detail page

### PR DESCRIPTION
**Pull request by ryugoo**
## Issue
- #176

## Overview (Required)
- Use "colorPrimaryDark" matching the primary color on the session detail page

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionDetailFragment.java [hunk 1]
```diff
@@ -85,6 +79,17 @@ private void initTheme() {
                     new ActivityManager.TaskDescription(viewModel.getSessionTitle(), null,
                             ContextCompat.getColor(activity, viewModel.getSessionVividColorResId()));
             activity.setTaskDescription(taskDescription);
+
+            // Change status bar scrim color
+            Window window = activity.getWindow();
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionDetailFragment.java]: I think it's not necessary 🤔 
`activity.setTheme(viewModel.getTopicThemeResId());` is enough to change the theme.
Actually, I could change the status color on DroidKaigi2016 app.
https://github.com/konifar/droidkaigi2016/blob/master/app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionDetailFragment.java#L61

**Comment by konifar** [comment]
Thanks for contribution!

File: app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionDetailFragment.java (hunk 1 above)
Comment (ryugoo) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionDetailFragment.java]: Thank you for your comment!
Certainly this code was unnecessary. Delete unnecessary code and push it.

**Comment by konifar** [comment]
LGTM!

**Pull request merged**